Backend/Data/KillerMoveTable.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Backend.Data.Struct;
+
+namespace Backend.Data;
+
+public class KillerMoveTable
+{
+
+    private const int SIZE = 2 * 64;
```

Comment:
Is this going to be mutated like 3/4/5 * 64 as well? If not, why the multiplication?

Reply:
It isn't going to be mutated but it is essentially a conversion from a 2D array to a 1D array. It's a compile-time constant so won't slow down the program via multiplication :)